### PR TITLE
fix issues of weekly meeting

### PR DESCRIPTION
**Pull request by sunzhaoyang**
### First-time contributors' checklist 

- [x ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [x ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: TOC.md [hunk 1]
```diff
@@ -42,32 +42,29 @@
...
-  + Advanced Migration
-    + [Overview](/data-migration/advanced-migration/overview.md)
+  + Advanced Migration To TiDB
+    + [Overview](/data-migration/advanced-migration/advanced-migration-overview.md)
     + [Filter binlog event](/data-migration/advanced-migration/binlog-filter.md)
     + [Work with GH-ost and PT-osc](/data-migration/advanced-migration/ghost-ptosc.md)
     + [Merge multiple tables and databases](/data-migration/advanced-migration/merge-db-table.md)
     + [Multiple tables downstream](/data-migration/advanced-migration/more-column.md)
-  + Case Example
-    + [Migrate from Aurora to TiDB](/data-migration/case/migrate-aurora-tidb.md)
```
Comment (Liuxiaozhen12) [inline TOC.md:69]: [Migrate from Aurora to TiDB] 这个 case 改到 Aurora 目录下了吗？没有找到呢。

**Review by Liuxiaozhen12 — APPROVED** [review]

File: TOC.md (hunk 1 above)
Comment (sunzhaoyang) [reply TOC.md:69]: 我暂时先删掉了，琢磨下怎么放。先弄个没有重名文件的demo给商务同学看

**Review by Liuxiaozhen12 — APPROVED** [review]

**Pull request merged**